modules/spring-config/src/main/java/org/mule/runtime/config/internal/BeanVertexWrapper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.internal;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * BeanVertexWrapper helps the comparison/equality check among non-proxy vs. proxy or proxy vs. proxy when building a dependency
+ * graph.
+ */
+public class BeanVertexWrapper {
+
+  final Object wrappedObject;
+  final String beanName;
+
+  public BeanVertexWrapper(String beanName, Object o) {
+    requireNonNull(o, "bean should not be null");
```

Comment:
should -> must

Reply:
Changed the name to BeanWrapper and the message is 'bean must not be null' now.